Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -195,7 +195,16 @@ class StatusTableViewController: UITableViewController, UIGestureRecognizerDeleg
             }
 
             reservoirVolume = dataManager.latestReservoirValue?.unitVolume
-
+			
+			// Remaining Volume HUD for x22 pumps
+			if reservoirVolume > 0 {
+				switch NSUserDefaults.standardUserDefaults().pumpModelNumber! {
```

Comment:
There's no guarantee this value exists, so force-unwrapping it could cause users to crash (especially first-time users or those who recently switched pump IDs).
It also should be made available to users with other pumps who might not have sentry packets configured for any reason, so it would be better in an else block on line 202/211:
```swift
} else if let capacity = dataManager.pumpState?.pumpModel?.reservoirCapacity {
    reservoirLevel = min(1, max(0, Double(reservoirVolume / capacity) / 100)
}
```

I know you expressed hesitation about adding reservoirCapacity to the rileylink_ios repo, but that doesn't prevent you from temporarily extending the `PumpModel` enum in this code base.

`Extensions/PumpModel.swift`
```swift
extension PumpModel {
    private var size: Int {
        return Int(rawValue)! / 100
    }

    var reservoirCapacity: Int {
        switch size {
        case 5:
            return 176
        case 7:
            return 300
        default:
            fatalError("Unknown reservoir capacity for PumpModel.\(self)")
        }
    }
}
```